src/v/kafka/server/usage_manager.cc

```diff
@@ -113,25 +113,32 @@ void usage_window::reset(ss::lowres_system_clock::time_point now) {
     u.bytes_cloud_storage = 0;
 }
 
+void usage_window::reset_to_nearest_interval(
```

Comment:
should this be called `clamp_to_low` `https://en.cppreference.com/w/cpp/algorithm/clamp` and be a generic template function?

Reply:
i guess fine to special case it.